packages/react-jss/src/JssProvider.js

```diff
@@ -20,85 +20,88 @@ type Props = {
   generateId?: GenerateId,
   classNamePrefix?: string,
   disableStylesGeneration?: boolean,
+  sheetOptions: StyleSheetFactoryOptions,
   children: Node
 }
 
 export default class JssProvider extends Component<Props> {
   static propTypes = {
-    ...propTypes,
+    registry: PropTypes.instanceOf(SheetsRegistry),
+    jss: PropTypes.shape({}),
```

Comment:
That should be Jss instance, right?

Reply:
yeah, but we don't export the Jss class

Reply:
can we export Jss as a type only?

Reply:
ah, we are in proptypes, so we would need a class here ... not sure whats best, mb we should export it, like we did with some others but not document.

Reply:
we could use jss instance to access the constructor as well, I think this should work: PropTypes.instanceOf(jss.constructor)